### PR TITLE
[9.0] [ADD] sale_revert_done

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
This module allows you to move back to state 'sale' a order that has been set to done.

### CODE REVIEW

File: sale_revert_done/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
```
Comment (elicoidal) [inline sale_revert_done/README.rst:28]: adapt

File: sale_revert_done/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Revert Done",
+    "summary": "Allow you to revert the state of a sale order from 'done' to "
```
Comment (elicoidal) [inline sale_revert_done/__openerp__.py]: be consistent with the README

File: sale_revert_done/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+               "'Sale Order' again.",
+    "version": "9.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
```
Comment (elicoidal) [inline sale_revert_done/__openerp__.py]: I'd rather have the OCA website by default

**Review by elicoidal — CHANGES_REQUESTED** [review]
some details

File: sale_revert_done/README.rst (hunk 1 above)
Comment (LoisRForgeFlow) [reply sale_revert_done/README.rst:28]: adapt to what?

Comment (angelmoya) [reply sale_revert_done/README.rst:28]: You need to put Report ID not Report Name, check

https://github.com/OCA/purchase-workflow/issues >> https://github.com/OCA/167/issues

Comment (LoisRForgeFlow) [reply sale_revert_done/README.rst:28]: That's right for the runbot link. But for the issues:
* https://github.com/OCA/purchase-workflow/issues: leads to the issues of that repo.
* https://github.com/OCA/167/issues: Leads to nowhere.

In adition, I tried to follow the template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

Comment (angelmoya) [reply sale_revert_done/README.rst:28]: Sorry, seems that https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst is deprecated, and @elicoidal and I were wrong

Comment (LoisRForgeFlow) [reply sale_revert_done/README.rst:28]: It's ok! the reviews are always welcomed  :smile:

**Review by elicoidal — APPROVED** [review]
Not sure whether I was wrong :p but the current URL suits me :)

**Comment by rafaelbn** [comment]
Hi @angelmoya  could you please make here last review? Thanks!

**Review by angelmoya — APPROVED** [review]
Code reviewed and Tested local, works fine

**Review by AaronHForgeFlow — APPROVED** [review]
tested on local + code review. Very useful module this one.     I guess runbot was killed because it's overloaded today

File: sale_revert_done/models/sale_order.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.one
```
Comment (serpentcs-dev1) [inline sale_revert_done/models/sale_order.py]: Better to use @api.multi and then iterate the self

**Review by serpentcs-dev1 — APPROVED** [review]
Code Review LGTM :+1:

File: sale_revert_done/README.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Sale Revert Done
+================
+
+This module extends the functionality of sales to allow you to set a sales
+order done back to state 'Sale Order'.
```
Comment (serpentcs-dev1) [inline sale_revert_done/README.rst]: I would write : 

`a done Sales Order back to ...`

**Comment by LoisRForgeFlow** [comment]
@darshan-serpent Can you check the latest changes?

**Comment by serpentcs-dev1** [comment]
LGTM :+1:

**Pull request merged**